packages/rocketchat-api/server/v1/chat.js

```diff
@@ -156,7 +156,7 @@ RocketChat.API.v1.addRoute('chat.sendMessage', { authRequired: true }, {
 		}
 
 		let message;
-		Meteor.runAsUser(this.userId, () => message = Meteor.call('sendMessage', this.bodyParams.message));
+		Meteor.runAsUser(this.userId, () => message = Meteor.call('sendMessage', this.bodyParams.message, 'RESTAPI'));
```

Comment:
We do not need to be adding origins and where things are coming from. Let's just always throw an error and if the web client needs to change to handle it, then we will need to change it.